sdk/src/manifest_store.rs

```diff
@@ -335,7 +335,9 @@ mod tests {
         assert!(manifest.time().is_some());
     }
 
-    #[actix::test]
+    #[cfg_attr(not(target_arch = "wasm32"), actix::test)]
```

Comment:
This is just so ugly, but it looks like what we have done in other places to make this work with wasm.

Reply:
Agreed it isn't pretty. _Very_ open to alternatives, this is just what I saw used elsewhere

Reply:
I was looking at a crate that would wrap complex macros like this to a single variable like #if_cfg(my_simple_var).  I will see if I can find it again.

Reply:
Yeah unfortunately the wasm_bindgen stuff is still pretty ugly, however we do use it in other places at the moment. IMO we should address fixing this in a future PR rather than switching everything over in this one.